doc/api/cli.md

```diff
@@ -234,7 +234,7 @@ Enable experimental JSON support for the ES Module loader.
 added: v9.0.0
 -->
 
-Specify the `module` of a custom [experimental ECMAScript Module loader][].
+Specify the `module` of a custom experimental [ECMAScript Module loader][].
```

Comment:
aren't loaders still experimental?

Reply:
Yes, it's just now part of the section status instead of the direct section title.